webapp/src/pages/Users/Users.css

```diff
@@ -2,13 +2,18 @@
   margin: 0;
   padding: 0;
   min-height: 100vh;
+  background-color: var(--color-background);
+  color: var(--color-light);
+}
+
+.users-wrapper {
+  width: 80%;
```

Comment:
This is just a comment about what is about to come when we implement screens, but since it Figma we are using grids to organize the content, maybe it worth to evaluate if we want to use them to organize content or just flex. For me, any of them is good.

Reply:
Good point! I vote for a grid layout to match the design in Figma. Flexbox is also a good choice.
Also, these pages are temporary. We will probably start from scratch when we start implementing the pages.